corehq/apps/locations/static/locations/ko/location_drilldown.async.js

```diff
@@ -143,7 +143,11 @@ function LocationModel(data, root, depth, func, withAllOption) {
 
   // helpers to account for the 'all' meta-entry
   this.num_children = ko.computed(function() {
-      return (this.children().length === 0 ? 0 : this.children().length - 1);
```

Comment:
Subtracting -1 excludes all option. It should be done only if there is all option.